REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia v0.6
-Compat v0.30.0
+julia 0.7
+Compat 0.30.0
```

Comment:
oops, I should have just had you delete Compat and all uses of it (since it's now not actually used). Do you mind?